mopidy_tidal/backend.py

```diff
@@ -45,6 +45,17 @@ def on_start(self):
         quality = self._config['tidal']['quality']
         logger.info("Connecting to TIDAL.. Quality = %s" % quality)
         config = Config(quality=Quality(quality))
+        client_id = self._config['tidal']['client_id']
+        client_secret = self._config['tidal']['client_secret']
+        if client_id and client_id.strip():
+            logger.info("Connecting to TIDAL.. Client ID read from config .. Client ID = %s" % client_id)
+            config.client_id=client_id
+            config.api_token=client_id
+            if client_secret and client_secret.strip():
+                logger.info("Connecting to TIDAL.. Client Secret read from config .. Client Secret = %s" % client_secret)
+                config.client_secret=client_secret
+            else:
+                logger.warn("Connecting to TIDAL.. client_id without client_secret will fail")
```

Comment:
This warning will never be displayed, if neither client_id nor client_secret is set in config

Reply:
i checked it ... if you set only client_id but no client_secret this warning will stop the extension to work

Reply:
but let me see if i can redo this part

Reply:
I misread your warning. I thought you wanted to have a warning if neither client_id/client_secret was set. 

But since you want to be able to fallback to using the built-in client_id/client_secret, I guess you can leave it as it is for now